Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/EquipmentRotation/ControllerTwo.mo

```diff
@@ -0,0 +1,284 @@
+block ControllerTwo
+  "Lead/lag or lead/standby equipment rotation controller for two devices or two groups of devices"
+
+  parameter Boolean lag = true
+    "true = lead/lag; false = lead/standby";
+
+  parameter Boolean continuous = false
+    "Continuous lead device operation"
+    annotation (Evaluate=true, Dialog(enable=not lag));
+
+  parameter Boolean initRoles[nDev] = {true, false}
+    "Initial roles: true = lead, false = lag/standby"
+    annotation (Evaluate=true, Dialog(tab="Advanced", group="Initiation"));
+
+  parameter Modelica.SIunits.Time stagingRuntime(
+    final displayUnit = "h") = 43200
+    "Staging runtime for each device"
+    annotation (Evaluate=true, Dialog(enable=not continuous));
+
+  parameter Buildings.Controls.OBC.CDL.Types.ZeroTime zerTim = Buildings.Controls.OBC.CDL.Types.ZeroTime.NY2019
+    "Enumeration for choosing how reference time (time = 0) should be defined"
+    annotation(Evaluate=true, Dialog(group="Calendar", enable=continuous));
+
+  parameter Integer yearRef(min=firstYear, max=lastYear) = 2019
+    "Year when time = 0, used if zerTim=Custom"
+    annotation(Evaluate=true, Dialog(group="Calendar", enable=zerTim==Buildings.Controls.OBC.CDL.Types.ZeroTime.Custom and continuous));
+
+  parameter Modelica.SIunits.Time offset = 0
+    "Offset that is added to 'time', may be used for computing time in a different time zone"
+    annotation(Evaluate=true, Dialog(group="Calendar", enable=continuous));
+
+  parameter Boolean weeInt = true
+    "Rotation is scheduled in: true = weekly intervals; false = daily intervals"
+    annotation(Dialog(group="Scheduler"));
+
+  parameter Integer houOfDay = 2
+    "Rotation hour of the day: 0 = midnight; 23 = 11pm"
+    annotation(Dialog(group="Scheduler"));
+
+  parameter Integer weeCou = 1 "Number of weeks"
+    annotation (Evaluate=true, Dialog(enable=weeInt, group="Scheduler"));
+
+  parameter Integer weekday = 1
+    "Rotation weekday, 1 = Monday, 7 = Sunday"
+    annotation (Evaluate=true, Dialog(enable=weeInt, group="Scheduler"));
+
+  parameter Integer dayCou = 1 "Number of days"
+    annotation (Evaluate=true, Dialog(enable=not weeInt, group="Scheduler"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uLeaStaSet if not continuous
+    "Lead device status setpoint"
+    annotation (Placement(transformation(extent={{-200,-20},{-160,20}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uLagStaSet if lag
+    "Lag device status setpoint"
+     annotation (Placement(transformation(extent={{-200,-100},{-160,-60}}),
+                          iconTransformation(extent={{-140,-20},{-100,20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uDevSta[nDev]
+    "Device proven ON status, where each index represents a physical device"
+    annotation (Placement(transformation(extent={{-200,60},{-160,100}}),
+      iconTransformation(extent={{-140,-80},{-100,-40}})));
```

Comment:
The output connectors should have the same size as input connectors (Diagram layer and icon layer). Just realized that other blocks have the same issue and will fix them in other PR.